linode/manager
# upcoming: [DI-20709] - CSS updates for widget level filters for ACLP

## Description 📝
Some simple CSS updates for reducing the height and font size of the widget level filter components.

## Changes  🔄

1. Updated the CSS of input props of the autocomplete components in widget level filters
2. Updated the height of the zoom icon component
3. Update the variant of typography to h2 in widget heading title.
4. Updated the multiselect components to not show placeholder if a item is selected
5. Updated the widget component by removing the divider

## Target release date 🗓️
16-09-2024

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-09-09 at 10 09 10 AM](https://github.com/user-attachments/assets/b08f555c-7267-4a78-97ec-59e079e67254)| ![Screenshot 2024-09-09 at 10 10 10 AM](https://github.com/user-attachments/assets/f674dd30-f6dd-4f9f-b9e5-b22ae2c16b35)|
|![Screenshot 2024-09-09 at 10 17 12 AM](https://github.com/user-attachments/assets/03604d5c-b5ec-4acd-8f86-abede9288b2f)|![Screenshot 2024-09-09 at 10 35 48 AM](https://github.com/user-attachments/assets/1c2501de-1f2f-44ca-9b30-4200e30078d7)|
|![Screenshot 2024-09-09 at 10 36 01 AM](https://github.com/user-attachments/assets/673d41b3-3886-4979-8979-2fbafe04bc24)||

## How to test 🧪

1. Login as mock user as some of endpoints are not production ready
2. Navigate to the monitor page
3. Select a linode dashboard and all required filters
6. You will be able to see the widgets with updated CSS.

## As an Author I have considered 🤔

*Check all that apply*
